src/plugin.tsx

```diff
@@ -104,11 +107,29 @@ export class KalturaLivePlugin implements OnMediaUnload, OnRegisterUI, OnMediaLo
             this._contribServices.overlayManager.remove(this._overlayItem);
             this._overlayItem = null;
         }
+        if (this._videoEnded) {
+            this._reloadVideo();
+        }
+    };
+
+    private _handleOnEnd = () => {
+        this._videoEnded = true;
+        this._setOffline();
+    };
+
+    private _reloadVideo = () => {
+        this._videoEnded = false;
+        this._player._detachMediaSource();
+        this._player._attachMediaSource();
```

Comment:
@semarche-kaltura consider changing this if we were able to really play. We need to understand better which of these calls is async/promise. Once we get this knowledge we can also try to hook the seek-to-live-edge that we are missing 
@eitanavgil  -  work with core team to expose this (vs call private methods) or to get commitment that this will not change.